src/main/java/com/untamedears/jukealert/model/actions/impl/BlockPlaceAction.java

```diff
@@ -33,8 +36,12 @@ public String getIdentifier() {
 
 	@Override
 	public IClickable getGUIRepresentation() {
-		// TODO Auto-generated method stub
-		return null;
+		ItemStack is = new ItemStack(getMaterial());
+		ItemMeta itemMeta = is.getItemMeta();
+		itemMeta.setDisplayName(ChatColor.GOLD + "Place");
```

Comment:
This might fail, because some materials can not be held in inventories. Try catch the ItemStack instanciation and if it fails, use stone, but add the right material as lore. Also note ItemAPI.addLore() in CivModCore, it keeps you from having to do that snippet every time.